zdm/tests/testlib.c

```diff
@@ -0,0 +1,6 @@
+#include <math.h>
+
+/* testlib.c */
+double f(int n, double *xx){
+    return xx[3]*exp(-0.5* pow((xx[0]-xx[1])/xx[2],2));
+}
```

Comment:
What is that stop sign symbol at the end - it that an ascii character? Just checking my editors will be able to make sense of it. Or is this something that github viewer is adding?
Anyway I highly approve of c++, I never used python until this job.

Reply:
must be a GitHub thing